src/components/TextareaField/TextareaField.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { generateSnapshots } from '@chanzuckerberg/story-utils';
+import { screen } from '@storybook/testing-library';
+
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import React from 'react';
+import { TextareaField } from './TextareaField';
+import * as stories from './TextareaField.stories';
+
+describe('<TextareaField />', () => {
+  generateSnapshots(stories);
+
+  it('can handle custom events when there is content', async () => {
+    const onChangeFn = jest.fn(() => console.log('test'));
```

Comment:
Don't think this needs the `console.log` anymore, since we can directly assert that the spy has been called.

IMO, console logging here just clutters up the console for little gain.

Reply:
Oh yah, I left that in when i was trying to verify the mock was being triggered. Meant to take that back out!

this PR was to remove the very thing you're talking about lol